addons/l10n_ke_edi_tremol/views/product_view.xml

```diff
@@ -22,7 +22,8 @@
         <field name="arch" type="xml">
             <xpath expr="//sheet" position="inside">
                 <group>
-                    <group name="KRA Item Code" string="KRA Item Code">
+                    <field name="fiscal_country_codes" invisible="1"/>
```

Comment:
Maybe add the field in `account` to avoid having to add it in multiple places in the future